src/main/groovy/com/github/spotbugs/snom/internal/SpotBugsRunner.java

```diff
@@ -108,6 +116,22 @@
     return args;
   }
 
+  private String createFileForAuxClasspath(Set<File> auxClasspathFiles) {
+    String auxClasspath =
+        auxClasspathFiles.stream().map(File::getAbsolutePath).collect(Collectors.joining("\n"));
+    try {
+      File tempFile = File.createTempFile("temp", "-spotbugs-auxclasspath");
+      FileWriter tempFileWriter = new FileWriter(tempFile);
+      tempFileWriter.write(auxClasspath);
+      tempFileWriter.close();
+      tempFile.deleteOnExit();
+      return tempFile.getAbsolutePath();
+    } catch (Exception e) {
+      // oops
+      throw new GradleException("Could not create auxiliary classpath file for SpotBugsTask");
```

Comment:
The original stack trace is lost.
You should pass as the second argument.

```suggestion
      throw new GradleException("Could not create auxiliary classpath file for SpotBugsTask", e);
```

Reply:
Thanks for the suggestion @wreulicke 

Should be ready again